.Rprofile

```diff
@@ -12,3 +12,58 @@ if (isTRUE(rownames(installed.packages(priority = "NA")) == "renv")) {
   renv::snapshot(prompt = FALSE) # create renv.lock
   message("Finished installing R package dependencies.")
 }
+
+# Configure locations of REMIND input data
+# These can be located in directories on the local machine, remote directories,
+# or default directories on the cluster.
+REMIND_repos_vars <- c('REMIND_repos_dirs',       # local directories
+                       'REMIND_repos_scp',        # scp connections
+                       'REMIND_repos_scp_user',   # scp user name
+                       'REMIND_repos_scp_key'     # ssh key path
+)
+
+# load repos variables into environment
+invisible(lapply(REMIND_repos_vars,
+                 function(x) { assign(x, Sys.getenv(x), pos = 1) }))
+use_cluster_defaults <- TRUE
+
+# add local directories, if any
+if ('' != get(REMIND_repos_vars[1])) {
+  foo <- unlist(
+    strsplit(REMIND_repos_dirs,
+             ifelse('Windows' == getElement(Sys.info(), 'sysname'),
+                    ';', ':'),
```

Comment:
```suggestion
             if ('Windows' == getElement(Sys.info(), 'sysname') ';' else ':',
```
> Further note that if(test) yes else no is much more efficient and often much preferable to ifelse(test, yes, no) whenever test is a simple true/false result, i.e., when length(test) == 1. 

https://rdrr.io/r/base/ifelse.html

Reply:
doesn't work:
```
Error: 50:65: unexpected string constant
49:     strsplit(remindReposDirs,
50:              if ('Windows' == getElement(Sys.info(), 'sysname') ';'
```

Reply:
There's a brace missing, it should be
```r
if ('Windows' == getElement(Sys.info(), 'sysname')) ';'
```

Reply:
doh, I thought it was simply not possible to have an expression there. That's what I get for not learning the formal R grammar properly…